LoRaEngine/LoraKeysManagerFacade/ RedisChannelPublisher.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using StackExchange.Redis;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Logging;
+    using LoRaTools;
+    using System.Text.Json;
+
+    public class RedisChannelPublisher : IChannelPublisher
+    {
+        private readonly ConnectionMultiplexer redis;
+        private readonly ISubscriber subscriber;
+        private readonly ILogger logger;
+
+        public RedisChannelPublisher(ConnectionMultiplexer redis, ILogger<RedisChannelPublisher> logger)
+        {
+            this.redis = redis;
+            this.logger = logger;
+            this.subscriber = this.redis.GetSubscriber();
+        }
+
+        public async Task PublishAsync(string channel, LnsRemoteCall lnsRemoteCall)
+        {
+            this.logger.LogDebug("Publishing message to channel '{Channel}'.", channel);
+            _ = await this.subscriber.PublishAsync(channel, JsonSerializer.Serialize(lnsRemoteCall));
+        }
```

Comment:
```suggestion
        private readonly ILogger logger;

        public RedisChannelPublisher(ConnectionMultiplexer redis, ILogger<RedisChannelPublisher> logger)
        {
            this.redis = redis;
            this.logger = logger;
        }

        public async Task PublishAsync(string channel, LnsRemoteCall lnsRemoteCall)
        {
            this.logger.LogDebug("Publishing message to channel '{Channel}'.", channel);
            _ = await this.redis.GetSubscriber().PublishAsync(channel, JsonSerializer.Serialize(lnsRemoteCall));
        }
```

We can simplify this by not storing the "GetSubscriber" result object (see [here](https://stackexchange.github.io/StackExchange.Redis/Basics.html#using-redis-pubsub) it is a "cheap pass-thru object that does not need to be stored")